mmv1/third_party/terraform/tests/resource_compute_instance_test.go.erb

```diff
@@ -4742,6 +4742,9 @@ resource "google_compute_instance" "foobar" {
   advanced_machine_features {
 	threads_per_core = 1
 	enable_nested_virtualization = true
+	<% unless version == 'ga' -%>
+	visible_core_count = 1
```

Comment:
Spacing here is a bit goofy in the generated test. Looks like extra tabs on the version guard lines are causing this